pkg/apis/tensorflow/v1alpha1/types.go

```diff
@@ -172,6 +172,9 @@ type ControllerConfig struct {
 
 	// Path to the file containing the grpc server source
 	GrpcServerFilePath string
+
+	// A flag for enabling gang scheduling by kube-arbitrator
+	EnableGangScheduling bool
```

Comment:
I suggest making this a command line flag; since its just a bool and not come complicated config like Accelerators.

Reply:
OK, I'll add the command line flag.